snap7/util.py

```diff
@@ -94,6 +94,10 @@
 import logging
 from snap7 import six
 import re
+import numpy as np
+import binascii
```

Comment:
This is unused as well - remove

Reply:
Ok. I only use it to test, which library I can use to parse the bytearray. 
Of course, these are to be deleted.